L1Trigger/Phase2L1ParticleFlow/interface/conifer.h

```diff
@@ -115,18 +115,19 @@ namespace conifer {
 
     std::vector<U> decision_function(std::vector<T> x) const {
       /* Do the prediction */
-      if (x.size() != n_features) {
-        throw cms::Exception("RuntimeError")
-            << "Conifer : Size of feature vector mismatches expected n_features" << std::endl;
-      }
+      //if (x.size() != n_features) {
+      //  throw cms::Exception("RuntimeError")
+      //      << "Conifer : Size of feature vector mismatches expected n_features" << std::endl;
+      //}
```

Comment:
one could throw a cms exception in CMSSW and a `std::` exception outside (or always a std:: one)

Reply:
Right, we noticed this trying to run the emulator outside of CMSSW context. I guess the usual `#ifdef CMSSW_GIT_HASH` should work. I think it's best to keep the runtime error checking, I will fix that.